lib/justifi/util.rb

```diff
@@ -78,5 +78,12 @@ def self.flatten_params_array(value, calculated_key)
       end
       result
     end
+
+    # Creates a computed signature that can verify the payload sent.
+    # The Justifi.client_secret is used to encrypt the payload
+    def self.compute_signature(params)
+      payload = params.keys.sort.map { |k| "#{k}#{params[k]}" }.join("")
```

Comment:
This is not quite the right logic for signing, we do not use the client secret. We will need a new signature key variable to use. Each webhook endpoint has it's own signature key. Also we put the detail to json and add the timestamp instead of mapping the key/values. : https://github.com/justifi-tech/lambdas/blob/main/lib/webhook_signer/src/app.rb#L36-L39